package.json

```diff
@@ -58,17 +55,16 @@
     "jest": "^26.4.2",
     "jest-styled-components": "^7.0.3",
     "lint-staged": "^10.4.0",
-    "prettier": "^2.1.2",
-    "react-docgen-typescript-loader": "^3.7.2",
-    "storybook-addon-jsx": "^7.3.4"
+    "prettier": "^2.0.5",
```

Comment:
Is there an issue with the latest prettier release that made you change this to an earlier version?

Reply:
Good catch. I was testing out a fix and forgot to revert it!